SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -352,7 +352,7 @@ UClass* USpatialClassInfoManager::GetClassByComponentId(Worker_ComponentId Compo
 	}
 	else
 	{
-		UE_LOG(LogSpatialClassInfoManager, Warning,
+		UE_LOG(LogSpatialClassInfoManager, Log,
```

Comment:
@joshuahuburn Adding you here to be aware and approve of this change, regarding a conversation we had in slack: https://improbable.slack.com/archives/G0179B73XM4/p1601390369082100